src/main/java/com/google/collegeplanner/data/Meeting.java

```diff
@@ -0,0 +1,151 @@
+  private ArrayList<DayOfWeek> days;
+  private String room;
+  private String building;
+  private int startTime;
+  private int endTime;
+
+  public Meeting(String days, String room, String building, 
+  int startTime, int endTime) throws ParseException {
+    this.days = new ArrayList<DayOfWeek>();
+    this.room = room;
+    this.building = building;
+    this.startTime = startTime;
+    this.endTime = endTime;
+
+    if (days.toUpperCase().contains("M")) {
+      this.days.add(DayOfWeek.MONDAY);
+    }
+    if (days.toUpperCase().contains("TU")) {
+      this.days.add(DayOfWeek.TUESDAY);
+    }
+    if (days.toUpperCase().contains("W")) {
+      this.days.add(DayOfWeek.WEDNESDAY);
+    }
+    if (days.toUpperCase().contains("TH")) {
+      this.days.add(DayofWeek.THURSDAY);
+    }
+    if (days.toUpperCase().contains("F")) {
+      this.days.add(DayOfWeek.FRIDAY);
+    }
+    if (days.toUpperCase().contains("SA") || days.toUpperCase().contains("SU")) {
+      throw new ParseException("Invalid Day, no Weekends on Academic Calendar");
+    }
+  }
+
+  public Meeting(JSONObject json) throws ParseException {
+    this((String) json.get("days"), (String) json.get("room"), (String) json.get("building"),
+        (int) json.get("start_time"), (int) json.get("end_time"));
+  }
+
+  public boolean conflictsWith(Meeting other) {
+    // Checks to see if the two meetings occur on the same day
+    // If not, no need to check the time as they cannot conflict
+    boolean canSkip = true;
+    for (String day : other.getDays()) {
+      if (days.contains(day)) {
+        canSkip = false;
+        break;
+      }
+    }
+
+    if (canSkip) {
+      return false;
+    }
+
+    if (startTime > other.getStartTime()) {
+      if (startTime >= other.getEndTime()) {
+        return false;
+      }
+    } else {
+      if (endTime <= other.getStartTime()) {
+        return false;
+      }
+    }
+
+    return true;
+  }
+
+  public ArrayList<DayOfWeek> getDays() {
+    return days;
+  }
+
+  public String getRoom() {
+    return room;
+  }
+
+  public String getBuilding() {
+    return building;
+  }
+
+  public int getStartTime() {
+    return startTime;
+  }
+
+  public int getEndTime() {
+    return endTime;
+  }
+
+  @Override
+  public String toString() {
+    String toString = "";
+    for (DayOfWeek str : days) {
+      toString += str + " ";
+    }
+
+    toString += "From " + Integer.toString(startTime) + " to " + Integer.toString(endTime);
+
+    return toString;
+  }
+
+  public JSONObject toJSON() {
+    JSONObject json = new JSONObject();
+    String daysString = "";
+
+    if(days.contain(DayOfWeek.MONDAY)) {
+      daysString += "M";
+    }
+    if(days.contain(DayOfWeek.TUESDAY)) {
+      daysString += "TU";
+    }
+    if(days.contain(DayOfWeek.WEDNESDAY)) {
+      daysString += "W";
+    }
+    if(days.contain(DayOfWeek.THURSDAY)) {
+      daysString += "TH";
+    }
+    if(days.contain(DayOfWeek.FRIDAY)) {
+      daysString += "F";
+    }
```

Comment:
missing saturday/sunday

Reply:
Saturday and sunday cannot be added into `days` because earlier in the code, if in the constructor the user tries to add saturday and sunday, an error is thrown.

Reply:
Let's add a comment here explaining that.